folding-schemes/src/folding/nova/cyclefold.rs

```diff
@@ -82,6 +86,87 @@ where
     }
 }
 
+// A more efficient version of `NonNativeFieldVar::to_constraint_field`
+fn nonnative_field_var_to_constraint_field<TargetField: PrimeField, BaseField: PrimeField>(
```

Comment:
This is great! How would you see to put this method in the `folding-schemes/src/folding/circuits/*` dir? Because probably we're going to use it in the other folding schemes.